debugger/src/dconfig.c

```diff
@@ -67,8 +67,13 @@ static GKeyFile *keyfile_project = NULL;
 static gboolean debug_config_loading = FALSE;
 
 /* saving thread staff */
+#if GTK_CHECK_VERSION(3, 0, 0)
+static GMutex change_config_mutex;
+static GCond cond;
+#else
```

Comment:
This is glib specific code and does not depend on gtk. So a check for the gtk version seems to be wrong here.

Reply:
I agree. I replaced the deprecated glib code with the new one.